feeds/src/components/listing/GridItem.tsx

```diff
@@ -54,13 +55,13 @@ export const GridItem: React.FC<Props> = ({
             </>
           )}
         </div>
-        {feed.sponsored && feed.sponsored.length > 0 && (
+        {sponsors.length > 0 && (
```

Comment:
This line breaks the tests as we show sponsors only when answers are fetched (`listingAnswer`)

Reply:
Ah nice catch, was typo on my side